erpnext/crm/doctype/opportunity/opportunity.py

```diff
@@ -268,6 +293,9 @@ def set_missing_values(source, target):
 				"opportunity_from": "quotation_to",
 				"opportunity_type": "order_type",
 				"name": "opportunity",
+				"applies_to_serial_no": "applies_to_serial_no",
+				"applies_to_vehicle": "applies_to_vehicle",
+				"sales_person": "service_advisor"
```

Comment:
Remove sales person mapping .. or map it into the sales team table instead

Reply:
But we don't have sales_team table in quotation